tools/build/configure.py

```diff
@@ -114,16 +114,37 @@ def write_config():
                     bazel_rc.write('build --action_env TF_CUDA_VERSION="10.1"\n')
                     bazel_rc.write('build --action_env TF_CUDNN_VERSION="7"\n')
             # Needed for tf rules
-            bazel_rc.write('build --experimental_repo_remote_exec\n')
+            bazel_rc.write("build --experimental_repo_remote_exec\n")
             # Enable platform specific config
-            bazel_rc.write('build --enable_platform_specific_config\n')
+            bazel_rc.write("build --enable_platform_specific_config\n")
             # Needed for GRPC build
             bazel_rc.write('build:macos --copt="-DGRPC_BAZEL_BUILD"\n')
             # Stay with 10.14 for macOS
             bazel_rc.write('build:macos --copt="-mmacosx-version-min=10.14"\n')
             bazel_rc.write('build:macos --linkopt="-mmacosx-version-min=10.14"\n')
             # MSVC (Windows): Standards-conformant preprocessor mode
             bazel_rc.write('build:windows --copt="/Zc:preprocessor"\n')
+            # Config for CI and release build
+            bazel_rc.write("build:ci --copt=-msse4.2\n")
+            bazel_rc.write("build:ci --copt=-mavx\n")
+            bazel_rc.write("build:ci --compilation_mode=opt\n")
+            bazel_rc.write("build:mac_ci --copt=-Wunguarded-availability\n")
```

Comment:
I think `--copt=-Wunguarded-availability` can be part of the `build:macos` as there is no impact even for normal build?
```
build:macos --copt=-Wunguarded-availability
```

Reply:
Done !

Reply:
This actually wil invalidate our `macOS` build cache. The `Bazel macOS` seems to be invalidated as well. However, it is only a temporary problem, we are fine.